interfaces/builtin/x11.go

```diff
@@ -60,14 +137,70 @@ socket AF_NETLINK - NETLINK_KOBJECT_UEVENT
 bind
 `
 
+type x11Interface struct{}
```

Comment:
Could you perhaps embed the common interface and override the functions that need non-default implementation?

Reply:
Done!